.vaunt/config.yaml

```diff
@@ -96,7 +96,6 @@ actions:
                     action: issue_comment
                     condition:
                         "created_at >= 01-09-2024 08:00:00 & created_at <= 01-12-2024 07:59:00
-                        & emojis in ['THUMBS_UP']
                         & reactors in ['rpaik', 'lilin90', 'qiancai', 'hfxsd', 'ran-huang', 'Oreoxmt','Yan-yyCL','dveeden']
```

Comment:
If you want to remove the requirement for a reaction entirely you should also remove the reactors. 

The only thing is that this will mean all comments on an issue with the bonus label will get the points. Not just a specific comment or user.